### ISSUE TITLE
Accessibility issue: Page should contain a level-one heading on /content-editor

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#180]
## What
An accessibility scan flagged the element `<html lang="en" dir="ltr" prefix="og: https://ogp.me/ns#" class="sticky-footer js" style="--scrollbar-width: 0px; --full-width-left-distance: -82px; --full-width-right-distance: -82px; --sidebar-top-margin: 1395.203125px;">` because page should contain a level-one heading. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/page-has-heading-one?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/957b6ad6-6e04-46ff-b3f0-68d5f355658d.png)

To fix this, ensure that the page, or at least one of its frames contains a level-one heading.

Specifically:

Fix all of the following:
-   Page must have a level-one heading

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
